src/nvofapi/nvofapi_vk_instance.cpp

```diff
@@ -27,21 +27,18 @@
 namespace dxvk {
 
     bool NvOFInstanceVk::Initialize() {
-        // For VK we cannot load winevulkan directly or we may break handle
-        // opacity.
-        m_library = LoadLibraryA("vulkan-1.dll");
-        if (!m_library) {
+        // For Vulkan we cannot load `winevulkan.dll` directly, or we may break handle opacity
+        m_vk = m_resourceFactory.CreateVulkan("vulkan-1.dll");
+        if (!m_vk->IsAvailable()) {
+            log::info("Initializing NVOFAPI failed: could not find usable Vulkan loader (or vulkan-1) module in the current process");
```

Comment:
And here `vulkan-1` is pretty much the same thing as Vulkan loader (unless it's Wine's shim for winevulkan but that's an implementation detail).